modules/seqexec/server/src/main/scala/seqexec/server/SeqexecFailure.scala

```diff
@@ -44,6 +46,8 @@ object SeqexecFailure {
   /** XMLRPC error while communicating with the GDS */
   final case class GDSXMLError(msg: String, url: Uri) extends SeqexecFailure
 
+  final case class SeqexecCombinedFailure(a: SeqexecFailure, b: SeqexecFailure) extends SeqexecFailure
```

Comment:
Another solution in cases like this is to use `NonEmptyList[SeqexecFailure]` which gives you a free semigroup, so you don't need the reified constructor for `|+|`. Just FYI. But this is fine.